daml-lf/encoder/BUILD.bazel

```diff
@@ -72,10 +80,82 @@ da_scala_binary(
     ],
 )
 
+filegroup(
+    name = "lf_since_1.0",
+    srcs = glob(["src/test/lf/since_1_0/**/*.lf"]),
+)
+
+filegroup(
+    name = "lf_since_1.1",
+    srcs = glob(["src/test/lf/since_1_1/**/*.lf"]),
+)
+
+filegroup(
+    name = "lf_since_1.3",
+    srcs = glob(["src/test/lf/since_1_3/**/*.lf"]),
+)
+
+filegroup(
+    name = "lf_since_1.dev",
+    srcs = glob(["src/test/lf/since_1_dev/**/*.lf"]),
+)
+
+genrule(
+    name = "testing-dar-1.0",
+    srcs = [":lf_since_1.0"],
+    outs = ["test-1.0.dar"],
+    cmd = "$(location :encoder_binary) $(SRCS) -o $@ -v 1.0",
+    tools = [":encoder_binary"],
+    visibility = [
+        "//daml-lf:__subpackages__",
+        "//language-support:__subpackages__",
+    ],
+)
+
+genrule(
+    name = "testing-dar-1.1",
+    srcs = [
+        ":lf_since_1.0",
+        ":lf_since_1.1",
+    ],
+    outs = ["test-1.1.dar"],
+    cmd = "$(location :encoder_binary) $(SRCS) -o $@ -v 1.1",
+    tools = [":encoder_binary"],
+    visibility = [
+        "//daml-lf:__subpackages__",
+        "//language-support:__subpackages__",
+    ],
+)
+
 genrule(
-    name = "assembler-test-dalf",
-    srcs = ["src/test/resource/Test.damllf"],
-    outs = ["Test.dalf"],
-    cmd = "$(location :encoder_binary) $(SRCS) $@ 1.dev",
+    name = "testing-dar-1.3",
+    srcs = [
+        ":lf_since_1.0",
+        ":lf_since_1.1",
+        ":lf_since_1.3",
+    ],
+    outs = ["test-1.3.dar"],
+    cmd = "$(location :encoder_binary) $(SRCS) -o $@ -v 1.3",
     tools = [":encoder_binary"],
+    visibility = [
+        "//daml-lf:__subpackages__",
+        "//language-support:__subpackages__",
+    ],
+)
+
+genrule(
+    name = "testing-dar-1.dev",
+    srcs = [
+        ":lf_since_1.0",
+        ":lf_since_1.1",
+        ":lf_since_1.3",
+        ":lf_since_1.dev",
+    ],
+    outs = ["test-1.dev.dar"],
+    cmd = "$(location :encoder_binary) $(SRCS) -o $@ -v 1.dev",
+    tools = [":encoder_binary"],
+    visibility = [
+        "//daml-lf:__subpackages__",
+        "//language-support:__subpackages__",
+    ],
```

Comment:
Can you write these `filegroup` and `genrule` calls with list comprehension[s]? See https://github.com/digital-asset/daml/blob/e0e3ce51c7b22ffc78e8f684ed4c0a413d850659/daml-lf/tests/BUILD.bazel for examples.

Reply:
fixed